### ISSUE TITLE
[windows] install paths with spaces cause get_stylesheet() to fail

### ISSUE THREAD

**Issue by zkamvar** [carpentries/pegboard#72]
I received a notification from a tester on Windows that their RStudio session had crashed when they tried to run the infrastructure with the following error:

```
xslt error: compilation error: file C:/Users/[user]/AppData/Local/Temp/[Rtemp]/file line 10 element import
xslt error: xsl:import : invalid URI reference C:/Users/[user]/OneDrive - [university name]/Documents/R/win-library/4.1/tinkr/extdata/xml2md_gfm.xsl
```

I believe the fix for this should be to normalize the path for the tinkr stylesheet in `get_stylesheet()`: 

https://github.com/carpentries/pegboard/blob/c20989219970bcd6fa48d08f7f6f47114da47443/R/get_stylesheet.R#L1-L14

**Comment by zkamvar** [comment]
Note: `xml2::url_escape()` may work here---it replaces slashes with escape codes, but on tests, it appears to be working.

**Comment by zkamvar** [comment]
It appears superficially similar to this bug: https://saxonica.plan.io/issues/2314. `xml2::url_escape()` does _not_ appear to work. 

We have this sheet because we wanted to import the {tinkr} stylesheet and add our own styles on top of that. To solve this, I think there are two solutions:

1. just use tinkr's stylesheet and skip/remove the dovetail stylesheet tests
2. copy tinkr's stylesheet to a temporary location and use that for import.

**Comment by zkamvar** [comment]
This was fixed in #72, I just never tagged it -_-
